src/content/docs/recipes/handling-dark-mode.mdx

```diff
@@ -10,38 +10,52 @@ Adding dark mode support to improve the user experience is now a common need on
 
 <Steps>
 
-1. Getting the current theme from local storage or system preference
+1. To Avoid FOUC, we need to load the script in the layout file, the `is:inline` attribute in the script tag.
```

Comment:
Can you link to an article explaining FOUC?

Reply:
Here's one https://en.wikipedia.org/wiki/Flash_of_unstyled_content